.eslintrc

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
For some reason we moved from .eslintrc to having the rules defined in package.json.... I can't remember why and I don't personally feel strongly one way or another. Just commenting to create a paper trail of moving back!